indico/web/forms/widgets.py

```diff
@@ -198,6 +198,42 @@ def __call__(self, field, **kwargs):
                                                      search_url=self.search_url, choices=getattr(field, 'choices', []))
 
 
+class LocationWidget(JinjaWidget):
+    """Renders a collection of fields to represent location"""
+
+    def __init__(self):
+        super(LocationWidget, self).__init__('forms/location_widget.html', single_line=True)
+
+    def __call__(self, field, **kwargs):
+        if Config.getInstance().getIsRoomBookingActive():
+            rooms = {loc.name: {'data': self.get_sorted_rooms(loc)} for loc in field.locations}
+            venues = {'data': [{'id': loc.id, 'name': loc.name} for loc in field.locations]}
+        else:
+            rooms = venues = {'data': []}
```

Comment:
mutable vs immutable ;)

`foo = bar = None` etc is fine because `foo = True` will leave `bar` alone, but with `foo = bar = []` a modification to any of those two variables will modify the other one too since they point to the same object